src/assets/scripts/game/GameController.js

```diff
@@ -338,10 +290,8 @@ export default class GameController {
      * @method update_pre
      */
     update_pre() {
-        const score = this.game_get_score();
-
-        if (score !== prop.game.last_score) {
-            this.updateScore(score);
+        if (this.game.score !== prop.game.last_score) {
```

Comment:
This was one of the things I was actually going to ask you, since I see a mix. I know the intent is to change all of these in here to `this` and remove `prop`, reference issue #43.